tools/flaky_test_monitor/common/config.go

```diff
@@ -0,0 +1,76 @@
+package common
+
+import (
+	"bufio"
+	"os"
+	"strconv"
+	"strings"
+)
+
+// Properties used in property file
+const FailureThresholdPercent = "failures_threshold_percent"
+const FailuresSliceMax = "failures_slice_max"
+
+const DurationThresholdSeconds = "duration_threshold_seconds"
+const DurationSliceMax = "duration_slice_max"
+
+// const Duration
+
+type Config struct {
+	FailureThresholdPercent float32
+	FailuresSliceMax        int
+
+	DurationThresholdSeconds float32
+	DurationSliceMax         int
+}
+
+func ReadProperties(directory string) Config {
+	// looks for properties file with specific name
+	propertyFile := AddTrailingSlash(directory) + "flaky-test-monitor.properties"
+
+	file, err := os.Open(propertyFile)
+
+	AssertNoError(err, "error opening property file")
+	defer file.Close()
+
+	properties := map[string]string{}
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		// read in line by line
+		line := scanner.Text()
+
+		// if line starts with # or //, treat that as a comment and skip it
+		if strings.HasPrefix(line, "#") || strings.HasPrefix(line, "//") {
+			continue
+		}
+
+		// check that line has "="
+		equalIndex := strings.Index(line, "=")
+		if equalIndex > 0 {
```

Comment:
Doesn't handle the case where there is multiple `=` signs, but for now, since we're mostly doing numerical values for config, this is probably fine.

Could consider using `strings.Split`